frontend/public/components/secrets/_create-secret.scss

```diff
@@ -20,11 +20,18 @@
   margin-bottom: 25px;
 }
 
-.co-create-secret-form__link--remove-entry + .co-create-image-secret__form {
-  padding-top: 25px;
+.co-create-secret-form__link--remove-entry {
+  & + .co-create-image-secret__form,
+  & + .co-create-generic-secret__form {
+    padding-top: 25px;
+  }
 }
 
-.co-create-image-secret__form-wrapper + .co-create-image-secret__form-wrapper {
+.co-create-secret__form-entry-wrapper + .co-create-secret__form-entry-wrapper {
   border-top: 1px solid #ccc;
   padding-top: 10px;
 }
+
+.co-create-generic-secret__form .co-file-dropzone__textarea {
+  min-height: 80px;
```

Comment:
Set the `min-height` since I didn't wanted to be less then the `80px` if user will mess with the height and so still readable.

eg:
![1](https://user-images.githubusercontent.com/1668218/45295275-655bad80-b4fe-11e8-9966-d00217a9d46f.png)

Reply:
> Maybe something like 1em so it shows at least one line?

1em ~ 12px ... I would suggest to set either 4em(two lines) or 6em(three lines).

Reply:
Just confirming that this only a problem if the user drags the textarea too small?

If we want to handle that, I think we should try to handle it generically instead of separately in every form. I would take this out of the PR and open a low severity issue.

Reply:
> Just confirming that this only a problem if the user drags the textarea too small?

yes

I added the `min-height` style here since the default `min-height` for the [textarea is set to 300px](https://github.com/openshift/console/blob/master/frontend/public/components/utils/_file-input.scss#L42). Thats way bigger that what we need for the key/value or image secret form, since there scould be more entries, but for SSH source secret its OK, since there is only one instance.
I would suggest to drop the `min-height` for the generic secret form and lower the 300px to 
 5em(three lines) or 7em(four lines) which could be a compromise for both forms

Reply:
OK I understand now. We might revisit the styles for the file input component later on.